mathoxide-lib/src/lib.rs

```diff
@@ -1,7 +1,10 @@
 #![feature(cell_filter_map)]
 #![feature(generic_associated_types)]
 
+extern crate num_traits;
```

Comment:
`extern crate` is not needed in most cases. see [here](https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html#no-more-extern-crate)